extensions/amp-date-countdown/1.0/test/test-date-countdown.js

```diff
@@ -0,0 +1,401 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {DateCountdown} from '../date-countdown';
+import {mount} from 'enzyme';
+
+describes.sandboxed('DateCountdown 1.0 preact component', {}, (env) => {
+  let sandbox;
+  let clock;
+
+  function render(data) {
+    return JSON.stringify(
+      data,
+      (key, value) => {
+        if (typeof value === 'number') {
+          return String(value);
+        }
+        return value;
+      },
+      4
+    );
+  }
+
+  beforeEach(() => {
+    sandbox = env.sandbox;
+    clock = sandbox.useFakeTimers(new Date('2018-01-01T08:00:00Z'));
+  });
+
+  afterEach(() => {
+    clock.runAll();
+  });
+
+  function syncPromise(response) {
+    return {
+      then(callback) {
+        callback(response);
+      },
+    };
+  }
+
+  it('should render as a div by default', () => {
+    const props = {
+      render,
+      datetime: Date.parse('2018-01-01T08:00:00Z'),
+    };
+    const jsx = <DateCountdown {...props} />;
+    const wrapper = mount(jsx);
```

Comment:
Inline to `consgt wrapper = mount(<DayeCountdown ...>)`

Reply:
Updated in latest revision!